packages/testsuite/cypress/support/commands.ts

```diff
@@ -103,6 +103,19 @@ Cypress.Commands.add("flip", (configurationFormId, attributeName, value) => {
   });
 });
 
+Cypress.Commands.add("getDefaultValue", (managementEndpoint, address, name) => {
```

Comment:
If this method is getting only boolean values, please rename it to make it more clear that the value should be boolean, e.g `getDefaultBooleanValue` or `getDefaultValueAsBoolean`. In case we'll find more use cases for this method, we could rename it to `getDefaultValue` (and also change the method signature)

Reply:
the `getDefaultBooleanValue` method created.